ocaml/libs/ezxenstore/core/dune

```diff
@@ -9,5 +9,6 @@
     (re_export xenstore)
     (re_export xenstore_transport)
     threads.posix
+    xapi-stdext-unix
```

Comment:
This needs to be added to `ezxenstore.opam.template` as well

Reply:
Added as a fixup, will squash once people had a chance to review the other commits.